lib/concepts/school_teacher/remove.rb

```diff
@@ -16,6 +16,7 @@ def call(school:, teacher_id:, token:)
       private
 
       def remove_teacher(school, teacher_id, token)
+        # TODO: This has not been implemented yet
```

Comment:
Just checking this is good to be here!

Reply:
I wanted to point this out for future us, since it wasn't obvious that the teacher list stuff was only stubbed